packages/dapp/src/components/Dashboard/NewsroomsListItem.tsx

```diff
@@ -23,57 +20,48 @@ interface ApplicationProgressData {
 }
 
 interface NewsroomListItemOwnProps {
-  listingAddress: EthAddress;
+  listing: any;
   applicationProgressData?: ApplicationProgressData;
 }
 
 const NewsroomsListItemGraphQL: React.FunctionComponent<NewsroomListItemOwnProps> = props => {
-  const { listingAddress, applicationProgressData } = props;
-  return (
-    <Query query={LISTING_QUERY} variables={{ addr: listingAddress }} pollInterval={10000}>
-      {({ loading, error, data }: any): JSX.Element => {
-        if (loading) {
-          return <LoadingMessage />;
-        }
-        if (error) {
-          console.error("Error querying listing", error);
-          return <ErrorLoadingDataMsg />;
-        }
-        if (!data.listing) {
-          if (applicationProgressData) {
-            const { tcrApplyTx } = applicationProgressData;
-            if (!tcrApplyTx || !tcrApplyTx.length) {
-              return (
-                <NoNewsrooms
-                  hasInProgressApplication={true}
-                  applyToRegistryURL={formatRoute(routes.APPLY_TO_REGISTRY)}
-                />
-              );
-            }
-          }
-          console.error("Error querying listing: no listing returned");
-          return <ErrorNotFoundMsg>We could not find the listing you were looking for.</ErrorNotFoundMsg>;
-        }
-        const listing = transformGraphQLDataIntoListing(data.listing, listingAddress);
-        const newsroom = transformGraphQLDataIntoNewsroom(data.listing, listingAddress);
-        const charterHeader = newsroom && newsroom.data && newsroom.data.charterHeader && newsroom.data.charterHeader;
+  const { listing, applicationProgressData } = props;
+
+  if (!listing) {
+    if (applicationProgressData) {
```

Comment:
This conditional branch will always return false because the only time this component is called, from `NewsroomsList`, we don't have `applicationProgressData` anymore. To make the refactor clearer I'd maybe remove this branch and the `applicationProgressData` prop. I'd also add a comment (or even update `ErrorNotFoundMsg`) to clarify that there might be an in-progress application.